tests/find/test-suite-1/test.array.js

```diff
@@ -79,6 +76,10 @@ testCases.push(function (dbType, context) {
         });
       });
 
+      // { name: 'James', _id: 'james',  favorites: ['Mario', 'Pokemon'], age: 20 },
+      // { name: 'Mary', _id: 'mary',  favorites: ['Pokemon'], age: 21 },
+      // { name: 'Link', _id: 'link', favorites: ['Zelda', 'Pokemon'], age: 22},
+      // { name: 'William', _id: 'william', favorites: ['Mario'], age: 23}
       it('should return docs match single value in array with defined index', function () {
```

Comment:
Since we checking which index this runs, can you convert this to use the explain method to check its actually using the correct index not just banking on the results. I think you might have to edit a few of these test for that.

Reply:
Hmm yeah, looking at lots of these tests they are specifying that they're using an index but we're not checking via explain. I suppose we should change them all? Did you want to hit that as part of your refactor, or should I do that here just for this file?